src/components/AddFarm/AddFarm.js

```diff
@@ -0,0 +1,169 @@
+import {
+  TextField, Button,
+} from '@material-ui/core';
+import React from 'react';
+import './AddFarm.css';
+
+// Airtable set-up here
+const Airtable = require('airtable');
+
+const base = new Airtable({ apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY })
+  .base(process.env.REACT_APP_AIRTABLE_BASE_KEY);
+
+export default class AddFarm extends React.Component {
```

Comment:
Refactor this to utilize function-based React programming instead of component-base React programming. Add React Hooks as well. 

See this component (Navbar.js) for an example:
https://github.com/lablueprint/farm2people-web-app/blob/master/src/components/Navigation/Navbar.js